public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "WebVR Studio",
+  "name": "Build WebVR experiences with WebVR Studio.",
```

Comment:
Found some reading on this. It does seem we'd want to change up these strings and make sure we have the files being referenced here.

https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/